compositeviews/compositeview01/es_settings.json

```diff
@@ -40,6 +40,7 @@
     }
   },
   "index": {
-    "number_of_shards": 1
+    "number_of_shards": 1,
+    "max_result_window": 500000
```

Comment:
@tobiasschweizer That's the only change needed - the other changes are from sairop-mods.

Reply:
I undid the SAIROP related changes.